backend/src/common/interceptors/profilingInterceptor.ts

```diff
@@ -47,4 +46,11 @@ export class ProfilingInterceptor implements NestInterceptor {
             `>> ${method} ${originUrl}\nAPI Call Duration: ${duration}ms\nProfiles: ${JSON.stringify(profilesWithDetails, null, 2)}\n\n`,
         );
     }
+
+    private async initProfiling(): Promise<void> {
+        await this.dataSource.query('SET profiling_history_size = 0');
+        await this.dataSource.query('SET profiling = 0');
+        await this.dataSource.query('SET profiling_history_size = 1000');
```

Comment:
`profiling_history_size`를 설정하는 것과 `profiling` 설정은 독립적이어서 여기서 `SET profiling = 0`을 할 필요는 없을 것 같아요.
또 mysql 공식 문서에 따르면 profiling_history_size의 최댓값은 100이라고 하네요.
> - [profiling_history_size](https://dev.mysql.com/doc/refman/8.4/en/server-system-variables.html#sysvar_profiling_history_size)
The number of statements for which to maintain profiling information if [profiling](https://dev.mysql.com/doc/refman/8.4/en/server-system-variables.html#sysvar_profiling) is enabled. The default value is 15. **_The maximum value is 100._** Setting the value to 0 effectively disables profiling. See [Section 15.7.7.32, “SHOW PROFILES Statement”](https://dev.mysql.com/doc/refman/8.4/en/show-profiles.html).
```suggestion
        await this.dataSource.query('SET profiling_history_size = 100');
```